### ISSUE TITLE
FEAT: Saved Queries import/export or external directory

### ISSUE THREAD

**Issue by OnionChild** [beekeeper-studio/beekeeper-studio#804]
**Details:**
Where are the "Saved" sql queries stored?  Can I add .sql files or other folders to the "Saved Queries" list?  I'd like to be able to use the same .sql file in Beekeeper as well as for another application

**Version Information [might be relevant to your issue]**
Operating System: MacOS 11.6
App Version [help -> about]: 2.1

**Screenshot / GIF [if appropriate]**:
![Screen Shot 2021-09-17 at 2 14 21 PM](https://user-images.githubusercontent.com/7296860/133854330-8129dd2d-2651-48de-bbbd-48be731e55d3.png)

**Comment by Zerotask** [comment]
It's probably stored in an internal db of beekeeper.
For your use case, an export & import of saved queries would be handy.

**Comment by OnionChild** [comment]
Okay.  If I understand correctly, it sounds like "Saved Queries" are hidden, so I can't actually do this at the moment.

**Comment by Zerotask** [comment]
It was an assumption and I told you a possible solution / better feature request for your use case.
Therefore I don't know why you close this feature request now.

**Comment by OnionChild** [comment]
Oh, I closed it because my question was answered.  But I suppose we can change it to a feature request

**Comment by rathboma** [comment]
Hey! Yes we're thinking about ways to do import/export or filesystem for queries, while not breaking our longer term plans with query management.

Actually fun workaround for exporting saved queries:
- Help -> add beekeeper's database
- Connect to Beekeeper's database
- right click favorite_queries table -> export

**Comment by psmgeelen** [comment]
I love what you guys are doing! Wouldn't it make more sense to be able to attach a working directory to be able to integrate it with projects? (Add some version control, and you would have a fully featured DB IDE!)

**Comment by anilsg** [comment]
I really like this idea of linking to a directory, either in `.config` or configurable so we can integrate it with our directory management. Then we can use our preferred tools on the files as well (e.g. vim), and there's no interface bloat.

It's the same for settings control in many apps. It's so frustrating when the settings GUI has faults or limitiations when if you just had a text config file you get ultimate ease, reliability and control.

**Comment by rathboma** [comment]
Yeah we need to re-think query and connection storage for sure. Soon.

For now - everything is in a SQLite database! So just query it with Beekeeper :-D

**Comment by anilsg** [comment]
> For now - everything is in a SQLite database! So just query it with Beekeeper :-D
Yes `/home/user/.config/beekeeper-studio/app.db` works.
You may need to right click to discover a "show hidden" setting for your particular file manager.

**Comment by rathboma** [comment]
Or just type `xdg-open /home/user/.config/beekeeper-studio/app.db`

**Comment by ZhaoKunLong** [comment]
Hello @rathboma, I was inspired by your answers and wanted to reach out for some help. 
I'm currently using Beekeeper Studio `v3.5.1`. 

Recently, I got a new computer and would like to continue using Beekeeper on it. However, when I tried to copy the program directly from my old computer, I found that all the saved queries were missing. I've been looking for a way to recover those valuable SQL queries.

After seeing your response, I immediately checked for the `/user/.config/beekeeper-studio/app.db` file, but since I was using an older version, I couldn't find it. 

Therefore, I would like to ask if you know where these SQL queries are typically stored in older versions (eg. `v 3.5.1`). Thank you

**Comment by rathboma** [comment]
Hey!

We're you using the snap version of the app by chance?

Matthew
====
Founder
